### DISCUSSION TITLE
No access to GitHub Copilot after successfully reapplying as a student

### DISCUSSION THREAD

**Discussion by zynth666** [community/community#67860] [GitHub Education] [answered]
### Hi everyone,

I can't get Copilot to work anymore. I have successfully reapplied as a student and I have access to the student campus etc. but Copilot won't work.

![grafik](https://github.com/community/community/assets/11150388/29af298d-a32e-4161-b9a5-a79f03b50856)

Any help would be greatly appreciated!

**Comment by busybox11** [comment, marked as answer]
You have to activate it from your settings. It will still mention the cost - but as soon as you try to register for it, it should prompt you to activate it for free with your Student benefits, as long as you have access to them.

**Reply by zynth666** [reply]
I went to https://github.com/features/copilot and clicked on Copilot for Individuals. And yeah then it registered me as a student again. So thank you. :D

**Reply by busybox11** [reply]
You're welcome!

**Reply by yashmittal26** [reply]
Thanks I was searching it for more than 1 hour, You saved my time, Thanks

**Comment by UmeshMehta1** [comment]
thank you
